src/components/Countries.js

```diff
@@ -0,0 +1,117 @@
+import React, { useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { Link } from 'react-router-dom';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faArrowAltCircleRight } from '@fortawesome/free-solid-svg-icons';
+import { getCountries } from '../redux/country/countries';
+import { getWorld } from '../redux/world/world';
+import logo from '../world.gif';
+
+const Countries = () => {
+  // For Country
+  const dispatch = useDispatch();
+  const countries = useSelector((state) => state.countriesReducer);
+  const [searchTerm, setSearchTerm] = useState('');
+  useEffect(() => {
+    if (countries.length === 0) dispatch(getCountries());
+  }, []);
+  // For World
+  const dipatchWorld = useDispatch();
+  const world = useSelector((state) => state.worldReducer);
+  useEffect(() => {
+    if (world.length === 0) dipatchWorld(getWorld());
+  }, []);
+  return (
+    <div>
+      <div className="row">
+        <div className="col">
+          <div className="card imgContainer mt-3">
+            <img src={logo} className="imgGif card-img" alt="logo" />
+            <div className="ontop card-img-overlay">
+              <div className="titleWorld card-title">
+                WORLD COVID 19
+              </div>
+
+              {
+                  world
+                    .map((item) => (
+                      <>
+                        <div className="subtitleWorld card-text">
+                          ACTIVE:
+                          {' '}
+                          {item.active.toLocaleString()}
+                        </div>
+                        <div className="stadisticsWorld card-text">
+                          DEATHS:
+                          {' '}
+                          {item.deaths.toLocaleString()}
+                        </div>
+                        <div className="subtitleWorld card-text">
+                          RECOVERIES:
+                          {' '}
+                          {item.recovered.toLocaleString()}
+                        </div>
+                        <div className="stadisticsWorld card-text">
+                          POPULATION:
+                          {' '}
+                          {item.population.toLocaleString()}
+                        </div>
+                      </>
+                    ))
+                }
```

Comment:
- [ ] When you use an iterator to generate a list of HTML elements, you should always give each element a unique key property. Keys help React identify which items have changed (added/removed/re-ordered). To give a unique identity to every element inside the array, a key is required. (The application has 2 console errors related to missing key properties, please check them)

Reply:
Hi Jasem,
I just realized your pull request doesn't follow the git-flow
You should have opened the pull request from development to the main branch (Sorry for noticing this after posting my review, I should have marked this request as invalid)
Since your main branch is empty you can remove it and rename your branches (I can help you with this, send me a message on slack)